egs2/aishell3/tts1/local/data.sh

```diff
@@ -69,7 +70,7 @@ if [ ${stage} -le 2 ] && [ ${stop_stage} -ge 2 ]; then
         scripts/audio/trim_silence.sh \
              --cmd "${train_cmd}" \
              --nj "${nj}" \
-             --fs 44100 \
+             --fs 24000 \
```

Comment:
For a double check, is the original data in 24k?

Reply:
No, it is 44.1k. Should I revert it to 44.1k?